scheduler/src/cook/kubernetes/api.clj

```diff
@@ -788,7 +788,8 @@
     (when (pos? gpus)
       (.putLimitsItem resources "nvidia.com/gpu" (double->quantity gpus))
       (.putRequestsItem resources "nvidia.com/gpu" (double->quantity gpus))
-      (add-node-selector pod-spec "cloud.google.com/gke-accelerator" gpu-model-requested))
+      (add-node-selector pod-spec "cloud.google.com/gke-accelerator" gpu-model-requested)
+      (add-node-selector pod-spec "gpu-count" gpus))
```

Comment:
Might be nice to have an explanatory comment above this line

Reply:
added

Reply:
I don't see the comment